test/test-clusters/src/main/java/org/elasticsearch/test/cluster/local/LocalSpecBuilder.java

```diff
@@ -131,6 +131,11 @@ interface LocalSpecBuilder<T extends LocalSpecBuilder<?>> {
      */
     T version(Version version);
 
+    /**
+     * Sets the version of Elasticsearch. Defaults to {@link Version#CURRENT}.
+     */
+    T version(String version);
```

Comment:
Added this as a convenience as many of our tests use `org.elasticsearch.Version` so then you have to disambiguate imports which is annoying. Plus, we almost always inject this into the build as a system property, which is a string to begin with.